pkg/workload/tpcc/tpcc.go

```diff
@@ -137,6 +141,38 @@ func (w *waitSetter) String() string {
 	}
 }
 
+type FileLoggerQueryTracer struct {
+	filePath string
+}
+
+func (t FileLoggerQueryTracer) TraceQueryStart(
+	ctx context.Context, conn *pgx.Conn, data pgx.TraceQueryStartData,
+) context.Context {
+	file, err := os.OpenFile(t.filePath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0666)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+	whitespacePattern := regexp.MustCompile(`\s+`)
+	fmt.Fprintf(file, "Query: %s\n", whitespacePattern.ReplaceAllString(strings.TrimSpace(data.SQL), " "))
+	return ctx
+}
+
+func (t FileLoggerQueryTracer) TraceQueryEnd(
+	ctx context.Context, conn *pgx.Conn, data pgx.TraceQueryEndData,
+) {
+	file, err := os.OpenFile(t.filePath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0666)
```

Comment:
instead of manually opening and closing an os file, could you use crdb logging infrastructure?

Reply:
Updated `FileLoggerQueryTracer` to match the logger used in the schema workload